pages/reference/supervisor/bandwidth-reduction.md

```diff
@@ -4,52 +4,47 @@ title: Reduce bandwidth usage
-|-------------------------------------------------------------------------|---------------------------------------------------------|--------------------------------------|
-| API poll (Once per 15min)                                               | 6650 Bytes per request                                  | BALENA_SUPERVISOR_POLL_INTERVAL      |
-| {{ $names.company.lower }} Supervisor update poll (Once every 24 hours) | 6693 Bytes per request                                  | Not configurable via variables       |
-| cloudlink enabled                                                       | 43 Bytes / second                                       | BALENA_SUPERVISOR_VPN_CONTROL        |
-| TCP check cost (When cloudlink disabled)                                | 47.36 Bytes / second                                    | BALENA_SUPERVISOR_CONNECTIVITY_CHECK |
-| Hardware metrics reporting                                              | ~66 Bytes every 10 seconds when there have been changes | BALENA_SUPERVISOR_HARDWARE_METRICS   |
+## Side Effects / Warnings
 
-Example minimum bandwidth settings
-----------------------------------
+**Cloudlink** Provides the ability to send instantaneous updates to the device. Turning off cloudlink has several downsides: (1) configuration and variable updates are delayed by the device API polling interval; (2) CLI access is disabled via the web terminal or SSH; (3) device's public URL is not accessible. To be clear, you can re-enable Cloudlink via API or console after turning it off. However you must wait for the next API poll from the device before the link is re-established.
 
-The following settings lead to data usage of approximately 1.3MB per month:
+**Device Metrics** [Device metrics][device-metrics] such as CPU temperature and memory usage won't be reported to the API if turned off, and so won't be displayed on the summary page of the device dashboard.
 
-* Disable `BALENA_SUPERVISOR_VPN_CONTROL`
-* Disable `BALENA_SUPERVISOR_CONNECTIVITY_CHECK`
-* Change `BALENA_SUPERVISOR_POLL_INTERVAL` to 24 hours (86400000 ms)
-* Disable `BALENA_SUPERVISOR_LOG_CONTROL`
-* Disable `BALENA_SUPERVISOR_HARDWARE_METRICS`
+**API Poll Interval** Defines the time interval between device checks for updates to the fleet, for example new code pushes, environment variables changes, or cloudlink control changes for the device. If Cloudlink is disabled, lengthening this interval increases the latency of propagating these updates to the device.
 
+**NetworkManager connectivity check** When enabled for devices with multiple network interfaces, NetworkManager uses routing metrics to avoid use of an interface that fails the check. This adaptability is crucial to retain connectivity. However for a single network interface, NetworkManager can provide only interface status from the check via DBus.
```

Comment:
```suggestion
**NetworkManager connectivity check**, when enabled for devices with multiple network interfaces, instructs NetworkManager to use routing metrics to avoid using an interface that fails the check. This adaptability is crucial to retain connectivity. However for a single network interface, NetworkManager can provide only interface status from the check via DBus.
```